core/src/main/java/org/infinispan/distribution/ch/ConsistentHash.java

```diff
@@ -172,4 +173,14 @@ default boolean isReplicated() {
     * Returns a string containing all the segments and their associated addresses.
     */
    String getRoutingTableAsString();
+
+   /**
+     * Writes this ConsistentHash to the specified scoped state.
+     *
+     * @param state the state to which this ConsistentHash will be written
+     */
+   /*default void toScopedState(ScopedPersistentState state) {
```

Comment:
I actually intended for the default method to be in the actual PR, as we're changing an interface in a minor release and I want to preserve backwards compatibility.